modules/users/client/components/ProfileViewBasics.component.js

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
+import { NamespacesConsumer } from 'react-i18next';
+import '@/modules/core/client/components/i18n';
 import PropTypes from 'prop-types';
-import moment from 'moment';
 
 export default function ProfileViewBasics({ profile }) {
-  return <>
+  return (<NamespacesConsumer ns="profile-view-basics">{ t => (<>
```

Comment:
I'd imagine we want to bundle all the translations of `user` module into one to keep thing simple?

Reply:
Changed to `user-profile`. Generally an easy fix.